### DISCUSSION TITLE
New Tab functionality like Arc Browser

### DISCUSSION THREAD

**Discussion by theaafofficial** [zen-browser/desktop#1355] [Ideas]
I recently switched from Arc Browser to Zen. I love Zen for its speed, security, and customization. I’m trying to find a way to type in the URL bar with Ctrl + T without opening a new tab, like in Arc Browser. Is this possible in Zen? If not, it would be a great feature to add.

**Comment by theaafofficial** [comment]
The closest thing I found is the Key Combination:
- Ctrl + L to open the URL bar, it'll open the URL bar of the current tab.
- Type your query or URL to open and hit Alt + Enter.

**Reply by DirectPass00** [reply]
thanks man!

**Comment by Muko-Tabi** [comment]
try [this](https://addons.mozilla.org/en-US/firefox/addon/vimium-c/) to see if it fit the function from Arc

**Comment by BenjaminJacober** [comment]
Same, would love this. Often times I'm in the flow of whatever I'm doing and I realize I didn't want to open a new tab, so I can just press escape and stop or maybe I still need information from my current tab for my next search. It also just makes sense to me, there should be no scenario where a user can even have an "empty" tab open. Empty referring to a tab that is on the home page. Or maybe it'd make sense to make it a mod/setting so that people that like to use the home page for shortcuts or something else can still have that functionality.
